cloudnet-modules/cloudnet-cloudperms/src/main/java/de/dytanic/cloudnet/ext/cloudperms/bukkit/BukkitCloudNetCloudPermissionsPlugin.java

```diff
@@ -110,16 +112,34 @@ private void addTeamEntry(Player target, Player all, IPermissionGroup permission
         if (team == null) team = all.getScoreboard().registerNewTeam(teamName);
 
         try {
+            Method method = team.getClass().getDeclaredMethod("setColor", ChatColor.class);
+            method.setAccessible(true);
+
+            String prefix = permissionGroup.getPrefix();
+            String color = permissionGroup.getColor();
+            String suffix = permissionGroup.getSuffix();
+
+            if (color != null && !color.isEmpty())
+                method.invoke(team, ChatColor.getByChar(color.replaceAll("&", "").replaceAll("§", "")));
```

Comment:
Please check if the ChatColor is null or not, and add brackets after the if-statement.